includes/html/pages/device/overview/ping.inc.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$perf = \DeviceCache::getPrimary()->perf;
+
+if ($perf->isNotEmpty()) {
+    $perf_url = Url('device') . '/device=' . DeviceCache::getPrimary()->device_id . '/tab=graphs/group=poller/';
+    echo '
+        <div class="row">
+        <div class="col-md-12">
+        <div class="panel panel-default panel-condensed">
+        <div class="panel-heading">
+        <a href="' . Url('device') . '/device=' . DeviceCache::getPrimary()->device_id . '/tab=graphs/group=poller/">
```

Comment:
You've set this url as the variable `$perf_url` above, why not use It here?

Reply:
Thanks for the review.
Must have forgotten to re-use the variable.
Fixed in the next commit.